waitress/app/utils.py

```diff
@@ -14,6 +14,26 @@ class UserRepository(object):
     A wrapper class for methods that update the list of users.
     """
 
+    @staticmethod
+    def add(user_type, username):
```

Comment:
`/users/update-users/` endpoint handles addition of new fellows so long as they are on Slack and in the fellows lagos channel